manifests/components/fluentd-es.jsonnet

```diff
@@ -21,7 +21,7 @@ local kube = import "../lib/kube.libsonnet";
 local kubecfg = import "kubecfg.libsonnet";
 local utils = import "../lib/utils.libsonnet";
 
-local FLUENTD_ES_IMAGE = "bitnami/fluentd:1.2.2-r22";
+local FLUENTD_ES_IMAGE = "bitnami/fluentd:1.2.6-r16";
```

Comment:
we'll want to backport this change. It would help the case if the version bump can be avoided